CONTRIBUTING.rst

```diff
@@ -135,9 +135,9 @@ Before you submit a pull request, check that it meets these guidelines:
    Github Actions.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
-   feature to the list in README.md.
-3. The pull request should work for Python 3.7, or later. Check
-   https://github.com/elfi-dev/elfi/actions/workflows/pytest.yml
+   feature to the list in README.rst.
```

Comment:
here `README.md` has been updated to `README.rst` but i think `README.md` was correct unless the intention was that the contributor is asked to update `index.rst` that has the same list in docs?

Reply:
Hmmm, this and the other issues you caught seem to indicate that I have updated the old versions of the documentation. Will fix them now.